docs/advanced_documentation/native-data-interface.md

```diff
@@ -141,7 +173,10 @@ The code below creates an array which is compatible with transformer input datas
 ```python
 from power_grid_model import ComponentType, DatasetType, power_grid_meta_data
 
-transformer = np.empty(shape=5, dtype=power_grid_meta_data[DatasetType.input][ComponentType.transformer]['dtype'])
+transformer_dtype = power_grid_meta_data[DatasetType.input][ComponentType.transformer]['dtype']
```

Comment:
shouldn't it be
```suggestion
transformer_dtype = power_grid_meta_data[DatasetType.input][ComponentType.transformer].dtype
```
?

Reply:
It can be accessed both ways. Was sticking to what was originally used.